pkgs/hover/default.nix

```diff
@@ -96,4 +97,8 @@ buildFHSUserEnv rec {
   ] ++ libs;
 
   runScript = "hover";
+
+  profile = ''
+    export SSL_CERT_FILE="${cacert}/etc/ssl/certs/ca-bundle.crt"
+  '';
```

Comment:
I think we should do this for hover, right?

Reply:
Yep, sorry, commited by mistake.